logfire-api/logfire_api/_internal/utils.pyi

```diff
@@ -1,5 +1,6 @@
 from _typeshed import Incomplete
 from collections.abc import Generator
+from logfire._internal.stack_info import is_user_code as is_user_code
```

Comment:
Why is the `as` needed?

Reply:
That's just what stubgen does, you can see it in lots of other places in the pyi files